openapi/v2/core-metadata.yaml

```diff
@@ -1243,6 +1252,55 @@ paths:
             application/json:
               schema:
                   $ref: '#/components/schemas/ErrorResponse'
+    put:
+      summary: "Allows updates to an existing device profile"
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                filename:
+                  type: string
+                  format: binary
+      responses:
+        '200':
```

Comment:
Are we only using the multi-part response (207) when the request could add/update multiple objects?